worker/addfmusite/createFMUTags.py

```diff
@@ -0,0 +1,72 @@
+import sys
+import os
+import json
+import uuid
+from pyfmi import load_fmu
+
+#1.0 setup the inputs
+fmupath = sys.argv[1]
+fmu_upload_name = sys.argv[2]
+jsonpath = sys.argv[3]
+fmu = load_fmu(fmupath)
+
+#2.0 get input/output variables from the FMU
+input_names = fmu.get_model_variables(causality = 2).keys()
+output_names = fmu.get_model_variables(causality = 3).keys()
+
+#print ("input names are: )))))): ", input_names )
+#print ("output names are: )))))): ", output_names )
+
+
+#3.0 add site tagging
+tags = []
+
+fmu_upload_name = os.path.basename(fmu_upload_name) # without directories
+fmu_upload_name = os.path.splitext(fmu_upload_name)[0] # without extension
+
+siteid = uuid.uuid4()
+sitetag = {
+    "dis": "s:%s" % fmu_upload_name,
+    "id": "r:%s" % siteid,
+    "site": "m:",
+    "simStatus": "s:Stopped",
+    "simType": "s:fmu",
+    "siteRef": "r:%s" % siteid
+}
+tags.append(sitetag)
+
+#4.0 add input tagging
+for var_input in input_names:
+    tag_input = {
+        "id": "r:%s" % uuid.uuid4(),
+        "dis": "s:%s" % var_input+':input',
```

Comment:
I think we should avoid adding ':input' and instead make the display name simply s:<var_input>. We can use a haystack tag like :writable to signal that it is an input.

Reply:
Thanks. Initially i used 'input'/'output' to distinguish the input_vars and output_vars. This is resolved by checking if "writable" tagging property existed in the haystack vars, which distinguishes the variables.